docs/usage.md

```diff
@@ -8,9 +8,17 @@
   * [Updating the pipeline](#updating-the-pipeline)
   * [Reproducibility](#reproducibility)
 * [Main arguments](#main-arguments)
+
+  Either (using a PRIDE Sample to data relation format file):
+  * [`--sdrf`](#--sdrf)
+  * [`--root_folder`](#--root_folder)
+
+  Or (using spectrum files and an OpenMS style experimental design):
   * [`--spectra`](#--spectra)
-  * [`--database`](#--database)
   * [`--exp_design`](#--exp_design)
+
+  And:
+  * [`--database`](#--database)
   * [`-profile`](#-profile)
```

Comment:
would it make sense to move the profile parameter to the nf core related ones

Reply:
I was not sure. On the one hand, yes it is a nf-core one. On the other hand, it is one of the most important main parameters and it would be nice to be a bit prominent.

Reply:
Considering that nf-core/rnaseq is the gold standard 

-> -profile is the very top parameter aka the most prominent one